tests/e2e/topup/topup.test.ts

```diff
@@ -2,48 +2,57 @@
 import test, { expect } from "@playwright/test";
 
 import { login } from "../../auth-helper";
-import { changeLanguage, ensureTextualElementExists, getInnerText, switchTab } from "../../utils";
+import { changeLanguage, ensureTextualElementExists, getInnerText, switchTab, wait } from "../../utils";
 
 test("Popup Page Should render", async ({ context }) => {
   const page = await login(context);
 
   // // see navigation works correctly
   await switchTab(page, "topup");
-
-  // RAMP TOPUP SHOULD WORK AS EXPECTED
-  // set amount to be transfered as 100 US Dollars, expect a positive value for expected SOL
-  await page.click("img[alt=Ramp]");
+  // ENSURE UI IS INTACT
+  await ensureTextualElementExists(page, "Select a Provider");
+  // MoonPay SHOULD WORK AS EXPECTED
+  // set amount to be transferred as 100 US Dollars, expect a positive value for expected SOL
+  await page.click("img[alt=moonpay]");
   await page.fill("input[type='number']", "100");
   const usdToSol100 = Number(await getInnerText(page, "#resCryptoAmt"));
   expect(usdToSol100).toBeGreaterThan(0);
 
-  // set amount to be transfered as 200 US Dollars, expect a positive value for expected SOL
+  // set amount to be transferred as 200 US Dollars, expect a positive value for expected SOL
   await page.fill("input[type='number']", "200");
   const usdToSol200 = Number(await getInnerText(page, "#resCryptoAmt"));
   expect(usdToSol200).toBeGreaterThan(0);
 
-  // ensure that on clicking save, it is refirected to Ramp Payment page having torus logo
-  await Promise.all([page.click("button:has-text('Top up')"), page.waitForEvent("popup")]);
-  await expect((await page.$$("img[alt=Ramp]")).length).toEqual(1);
+  // ensure that on clicking Top up, it is redirected to MoonPay Payment page
+  const [page2] = await Promise.all([page.waitForEvent("popup"), page.click("button:has-text('Top up')")]);
+  // closing moonpay page, will show an error on top up page
+  page2.close();
+  await wait(500);
+  await ensureTextualElementExists(page, "Transaction could not complete.");
 });
 
```

Comment:
Add test for changing of `fiat` currency or `crypto` currency also changes the value as expected.